nuxeo-drive-core/src/main/java/org/nuxeo/drive/service/impl/NuxeoDriveManagerImpl.java

```diff
@@ -158,7 +150,9 @@ public void invalidateCollectionSyncRootMemberCache(String userName) {
 
     @Override
     public void invalidateCollectionSyncRootMemberCache() {
-        log.debug("Invalidating collection sync root member cache for all users");
+        if (log.isDebugEnabled()) {
```

Comment:
checking `log.isDebugEnabled()` is useful when you have computation before logging